CME/admin/components/FrontMatter/Edit.php

```diff
@@ -305,6 +305,21 @@ protected function getSavedMessagePrimaryContent()
 	// }}}
 
 	// build phase
+	// {{{ protected function loadDBData()
+
+	protected function loadDBData()
+	{
+		parent::loadDBData();
+		$this->ui->getWidget('providers')->values = SwatDB::queryColumn(
```

Comment:
This would be better done with:
```
foreach ($this->getObject()->providers as $provider) {
   $this->ui->getWidget('providers')->values[] = $provider->id;
}
```